File: f77/build10/diffeq.f
C
      Subroutine Differential_Equations
C
C     This routine calls the routines that evaluate the state derivatives
C
      Call Actuator
      Call Aerodynamics
      Call Mass_Data
      Call Kinematics
      Call Target
      Call Seeker
      Call Gyro
      Call Flight_Computer
C
      Return
      End

File: compcert/build10/diffeq.c
#include "diffeq.h"
#include "actuator.h"
#include "aero.h"
#include "airframe.h"
#include "fcomp.h"
#include "gyro.h"
#include "kmtc.h"
#include "mass.h"
#include "seeker.h"
#include "target.h"
void differential_equations() {
  // This routine calls the routines that evaluate the state derivatives
  actuator();
  aerodynamics();
  mass_data();
  kinematics();
  target();
  seeker();
  gyro();
  flight_computer();
}
